File: dataset/output-bk/MISTRAL_codestral-latest/Task53_MISTRAL_codestral-latest.cpp
// C++
#include <iostream>
#include <unordered_map>
#include <uuid/uuid.h>

class SessionManager {
private:
    std::unordered_map<std::string, std::string> sessions;

public:
    std::string createSession(const std::string& userId) {
        uuid_t uuid;
        uuid_generate(uuid);
        char sessionId[37];
        uuid_unparse(uuid, sessionId);
        sessions[sessionId] = userId;
        return sessionId;
    }

    std::string getUserFromSession(const std::string& sessionId) {
        return sessions[sessionId];
    }

    void endSession(const std::string& sessionId) {
        sessions.erase(sessionId);
    }
};

File: dataset/output-bk/MISTRAL_codestral-latest/Task53_MISTRAL_codestral-latest.java
// Java
import java.util.HashMap;
import java.util.Map;
import java.util.UUID;

public class SessionManager {
    private Map<String, String> sessions = new HashMap<>();

    public String createSession(String userId) {
        String sessionId = UUID.randomUUID().toString();
        sessions.put(sessionId, userId);
        return sessionId;
    }

    public String getUserFromSession(String sessionId) {
        return sessions.get(sessionId);
    }

    public void endSession(String sessionId) {
        sessions.remove(sessionId);
    }
}